patch/views_bulk_operations_patch_1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/src/Service/ViewsBulkOperationsViewData.php b/src/Service/ViewsBulkOperationsViewData.php
+index d72ae49..0c9e274 100644
+--- a/src/Service/ViewsBulkOperationsViewData.php
++++ b/src/Service/ViewsBulkOperationsViewData.php
+@@ -220,7 +220,7 @@ class ViewsBulkOperationsViewData implements ViewsBulkOperationsViewDataInterfac
+       $total_results = $view->total_rows;
+     }
+ 
+-    if (!empty($pager_options)) {
++    if (!empty($pager_options) && !$pager_options['items_per_page'] === 0) {
+       $this->pagerManager->createPager($pager_options['total_items'], $pager_options['items_per_page'], $pager_options['id']);
+     }
+ 
```

Comment:
It this a custom made patch? Should it be mentioned in the readme?

Reply:
If not custom it should be install in compose with patch from DO.

Reply:
Added by @tuj in oct. 2020, so defiantly custom made. Documented in Readme best I can.